music21/noteworthy/translate.py

```diff
@@ -840,6 +972,61 @@ def createLyrics(self, attributes):
                         ll = ' - '
                     lyrics.append(ll)
         return lyrics
+    def createStaff(self, attributes):
+        self.createPart()
+        self.currentKey = key.KeySignature(0)
+        self.activeAccidentals = {}
+        self.lyrics = []
+        self.lyricPosition = 0
+        self.currentPart.partName = attributes.get('Name')
+        self.currentPart.partAbbreviation = attributes.get('Name')
+
+    def createStaffInstrument(self, attributes):
+        patch = int(attributes.get('Patch', 0))
+        try:
+            staffInstru = instrument.instrumentFromMidiProgram(patch)
+            readTranspo = int(attributes.get('Trans', 1))
+            self.currentTransposition = 12 * readTranspo // 12
+            staffInstru.transposition = interval.Interval(readTranspo)
+            self.currentPart.append(staffInstru)
+        except InstrumentException:
+            for p in range(34, 81):
```

Comment:
I don't think we should be creating 47 Instrument objects. Can we create one `UnpitchedPercussion` object?